docker/tensorflow-aarch64/cpu_info.sh

```diff
@@ -104,6 +110,26 @@ function set_target {
       eigen_l2_cache="1024*1024"
       eigen_l3_cache=
     ;;
+    neoverse-v1 )
+      cpu="neoverse-v1"
+      tune="neoverse-n1"
```

Comment:
Although the 'tune' build-arg isn't currently used during the build, it would make sense to set it to `neoverse-v1` here to avoid confusion.